service/app/util/RequestValidator.java

```diff
@@ -1088,4 +1095,49 @@ public static void validateUpdateEvent(Request contentRequestDto) {
       }
     }
   }
+
+  public static Boolean isProgram(String contentId) {
```

Comment:
refer CourseEnrolmentActor.getCourseContent method -- you need to do the same instead of calling the read API yourself...